sdk/communication/communication-common/src/identifierModels.ts

```diff
@@ -206,6 +257,26 @@ export const getIdentifierRawId = (identifier: CommunicationIdentifier): string
       }
       return `8:orgid:${microsoftTeamsUserId}`;
     }
+    case "microsoftBot": {
```

Comment:
in js is it possible to extract all duplicated hardcoded string values to the const?

Reply:
@AikoBB please see the similar discussion we've already had here https://github.com/Azure/azure-sdk-for-js/pull/23504#discussion_r1003093728